apps/native-ui-storybook/components/Alert/Variants.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Center, Alert, AlertIcon, AlertText } from '@utilitywarehouse/native-ui';
+
+import {
+  WarningMediumContainedIcon,
+  TickMediumContainedIcon,
+  CrossMediumContainedIcon,
+  InformationMediumContainedIcon,
+} from '@utilitywarehouse/react-native-icons';
+
+const AlertVariants = () => {
+  return (
+    <Center>
+      <Alert colorScheme="info" mb={4} gap="$3">
+        <AlertIcon as={InformationMediumContainedIcon} />
+        <AlertText>Unlock the power of knowledge with the following information.</AlertText>
+      </Alert>
+      <Alert colorScheme="success" mb={4} gap="$3">
```

Comment:
question about this: should we add the margin button to each child or just a gap to the main container?

Reply:
Good spot, I'd copied this over from the gluestack storybook but that's a good suggestion. Done!